md/multi-tenancy-and-tenant-configuration.md

```diff
@@ -55,8 +55,95 @@ Therefore, as long as you have not logged in to the new tenant, the login page w
+
+Start the tomcat
+##### Login
+
+    $ curl -v -c saved_cookies.txt -X POST --url 'http://localhost:8080/bonita/platformloginservice' \
+    --header 'Content-Type: application/x-www-form-urlencoded; charset=utf-8' \
+    -d 'password=platform&redirect=false&username=platformAdmin' -O /dev/null
+
+    * Connected to localhost (127.0.0.1) port 8080 (#0)
+    > POST /bonita/platformloginservice HTTP/1.1
+
+    < HTTP/1.1 200 OK
+
+    < Set-Cookie: JSESSIONID=46EF8A05819B6C268EE700F3C3FC939A; Path=/bonita/; HttpOnly
+
+    < Set-Cookie: X-Bonita-API-Token=a94cbf84-6b71-409a-981f-f91b17466929; Path=/
+
+The response to this REST API call (HTTP) generates 2 cookies, which must be transfered with each subsequent calls.
+One of the cookie is `X-Bonita-API-Token`.
+
+Note that the security against CSRF attacks is enabled by default for all fresh installations; the subsequence REST API calls using DELETE, POST, or PUT HTTP methods must define the `X-Bonita-API-Token` header, with the value transmitted via the associated cookie.
 
+The cookies had been saved on the disk, in the `saved_cookies.txt` file:
+
+    $ cat saved_cookies.txt 
+    
+    #HttpOnly_localhost	FALSE	/bonita/	FALSE	0	JSESSIONID	46EF8A05819B6C268EE700F3C3FC939A
+    localhost	FALSE	/	FALSE	0	X-Bonita-API-Token	a94cbf84-6b71-409a-981f-f91b17466929
+##### Create the new tenaant
+
+    $ curl -b saved_cookies.txt -X POST 'http://localhost:8080/bonita/API/platform/tenant' \
+    -H "Content-Type: application/json" \
+    -H 'X-Bonita-API-Token: a94cbf84-6b71-409a-981f-f91b17466929' \
+    -d '{"name":"MyTenant", "description":"My tenant", "username":"install", "password":"install"}'
+
+    {
+      "id": "101",
+      "creation": "2017-06-09 15:11:01.191",
+      "icon": "/default.png",
+      "username": "",
+      "description": "My tenant",
+      "name": "MyTenant",
+      "state": "DEACTIVATED",
+      "password": ""
+    }
+The new tenant has the id `101` and its state is `DEACTIVATED`
+##### Activate the tenant with id `101`
+
+    $ curl -v -b saved_cookies.txt -X PUT 'http://localhost:8080/bonita/API/platform/tenant/101' \
+    -H "Content-Type: application/json" \
+    -H 'X-Bonita-API-Token: a94cbf84-6b71-409a-981f-f91b17466929' \
+    -d '{"state":"ACTIVATED"}'
+    
+    * Connected to localhost (127.0.0.1) port 8080 (#0)
+    
+    < HTTP/1.1 200 OK
+
+    $ curl -b saved_cookies.txt -X GET 'http://localhost:8080/bonita/API/platform/tenant/101' 
+    
+    {
+      "password": "",
+      "name": "MyTenant",
+      "icon": "/default.png",
+      "description": "My tenant",
+      "id": "101",
+      "state": "ACTIVATED",
+      "creation": "2017-06-09 15:11:01.191",
+      "username": ""
+    }
+##### Logout
+
+    $ curl -v -b saved_cookies.txt -X GET --url 'http://localhost:8080/bonita/platformlogoutservice?redirect=false'
 ### Java PlatformAPI
+This solution can be used when the portal is not needed: it creates the tenant at engine side only.
```

Comment:
I think we should either remove the "at" or say "at engine level only" instead of "side"